.github/workflows/docker-bases.yml

```diff
@@ -0,0 +1,90 @@
+name: Build base compilers docker images
+
+on:
+  push:
+    paths:
+      - './docker/Dockerfile.nvidia'
+      - './docker/Dockerfile.cpu'
+      - './docker/Dockerfile.amd'
+  workflow_dispatch:
+    inputs:
+      tags:
+        description: 'Build compiler bases'
+  schedule:
+    # Run once a week
+    - cron: "0 13 * * 1"
+
+jobs:
+  deploy-docker-bases:
+    name: ${{ matrix.tag }}
+    runs-on: ${{ matrix.runner }}
+    env:
+      DOCKER_BUILDKIT: "1"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - tag: 'devitocodes/bases:nvidia-nvc'
+            arch: 'arch=nvc'
+            version: ''
+            dockerfile: './docker/Dockerfile.nvidia'
+            runner: ["self-hosted", "gpu", "openacc"]
+
+          - tag: 'devitocodes/bases:nvidia-clang'
+            arch: 'arch=clang'
+            version: 'ver=nvhpc-21-11'
```

Comment:
Considering the blood that has been spilt, perhaps add a comment pointing to the compatibility table of llvm and nvhpc?